autoprotocol/protocol.py

```diff
@@ -5034,7 +5034,7 @@ def image_plate(self, ref, mode, dataref):
         """
         return self._append_and_return(ImagePlate(ref, mode, dataref))
 
-    def provision(self, resource_id, dests, volumes):
+    def provision(self, resource_id, dests, volumes=None, amounts=None):
```

Comment:
would prefer to swap `volumes` and `amounts` positions, so it'll allow someone to specify mass as the third positional arg